### PR TITLE
[onecc-docker] Revisit onecc-docker command

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates `onecc-doecker` command.
- Change ubuntu version to 20.04
- Update python version requirement

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: compiler/onecc-docker/debian/control [hunk 1]
```diff
@@ -9,5 +9,5 @@ Homepage: https://github.com/Samsung/ONE
 Package: onecc-docker
 Architecture: amd64
 Multi-Arch: foreign
-Depends: ${misc:Depends}, ${shlibs:Depends}, python3.8
+Depends: ${misc:Depends}, ${shlibs:Depends}, python3
```
Comment (dayo09) [inline compiler/onecc-docker/debian/control:12]: :+1:

Comment (dayo09) [reply compiler/onecc-docker/debian/control:12]: Should the directory name be `compiler/onecc-docker/focal/control`?

**Review by jyoungyun — APPROVED** [review]
LGTM

File: compiler/onecc-docker/debian/control (hunk 1 above)
Comment (jyoungyun) [reply compiler/onecc-docker/debian/control:12]: @dayo09 This file is located in `onecc-docker/debian/control`. It does not have OS version name in the directory path. :)

Comment (dayo09) [reply compiler/onecc-docker/debian/control:12]: Oh, my mistake. I read 'debian' as  'bionic'. :sweat:

**Pull request merged**